src/main/java/com/bookstore/bookstoreapi/data/service/CustomerService

```diff
@@ -0,0 +1,36 @@
+package com.bookstore.bookstoreapi.service;
+
+import com.bookstore.bookstoreapi.data.entity.Customer;
+import com.bookstore.bookstoreapi.data.repository.CustomerRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CustomerService {
```

Comment:
Same as before:

there's no reason to create additional getters for these methods, they come pre-packaged and are fairly descriptive.